app/views/inthenews/home.html.erb

```diff
@@ -33,10 +49,6 @@
         </div>
       <% end %>
     </div>
-
-  </div>
-
-  <div class="col-md-6">
-    <%= render "partials/map_africa" %>
+    <%# render "partials/map_africa" %>
```

Comment:
Can we remove this since it should be trivial to add back by copying from the other page template?